apps/web/src/components/forms/status-page-form.tsx

```diff
@@ -20,21 +21,14 @@ import { Input } from "@/components/ui/input";
 import { Checkbox } from "../ui/checkbox";
 
 // REMINDER: only use the props you need!
-const schema = insertPageSchema
-  .pick({ title: true, slug: true, description: true })
-  .merge(
-    z.object({
-      monitors: z.string().array().optional(), // HOW TO PASS
-    }),
-  );
```

Comment:
I had struggles with `react-hook-form` when we have more props inside of the resolver. It would resolve the form to no be submitted because of validation. Therefore, only the props used where extracted.